samples/dotnet/ListTrigger.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Extensions.Logging;
```

Comment:
Is this trigger still used? I think this was your initial implementation which you have since moved away from right?

Reply:
Yes, this will also be removed

Reply:
Yes, it will be removed